docs/spark/how-to-guides/streaming.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Process streaming data with .NET for Apache Spark
+description: Learn how to use .NET for Apache Spark for Spark Structured Streaming.
+ms.date: 11/14/2019
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Process streaming data with .NET for Apache Spark
+
+This article teaches you how to invoke Spark Structured Streaming using .NET for Apache Spark. Spark Structured Streaming is Apache Spark's support for processing real-time data streams. Stream processing means analyzing live date as it's being produced.
+
+There are four full stream processing examples on GitHub:
+
+* [StructuredNetworkWordCount.cs](https://github.com/dotnet/spark/blob/master/examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/StructuredNetworkWordCount.cs): word count on data streamed from any source
+* [StructuredNetworkWordCountWindowed.cs](https://github.com/dotnet/spark/blob/master/examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/StructuredNetworkWordCountWindowed.cs): word count on data with windowing logic
+* [StructuredKafkaWordCount.cs](https://github.com/dotnet/spark/blob/master/examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/StructuredKafkaWordCount.cs): word count on data streamed from Kafka
+* [StructuredNetworkCharacterCount.cs](https://github.com/dotnet/spark/blob/master/examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/StructuredNetworkCharacterCount.cs): uses user-defined functions and stream processing to count the number of characters in each string read from a stream
+
+While the following steps apply to most stream processing apps, there are some specific references to the [StructuredNetworkCharacterCount.cs](https://github.com/dotnet/spark/blob/master/examples/Microsoft.Spark.CSharp.Examples/Sql/Streaming/StructuredNetworkCharacterCount.cs) example.
+
```

Comment:
I noticed in the GitHub batch processing doc, there was a mention of downloading the Microsoft.Spark nuget package, the `using` statements at the top of the program, and creating the `Run` method from `IExample`. Do we want to include those some explanations in this doc as well?